R/zchunk_L202.Ccoef.R

```diff
@@ -1,24 +1,26 @@
 #' module_energy_L202.Ccoef
 #'
-#' Briefly describe what this chunk does.
+#' This chunk creates the file of CO2 coefficients for primary energy sources.
```

Comment:
What is a CO2 coefficient? Maybe say something like "Create the file of CO2 coefficient (<definition>) for primary energy sources."